assets/js/main.js

```diff
@@ -25,6 +25,42 @@ function handleTopNavAnimation() {
  * SmoothScroll
 */
 
+$('a[href*="#"]')
```

Comment:
See comment on CSS file. I think, the version number needs to be changed for this also